src/mainwindow.cpp

```diff
@@ -211,7 +210,7 @@ void MainWindow::run(int index)
     connect(tmp, SIGNAL(runKilled(int)), this, SLOT(onRunKilled(int)));
     tmp->run(tmpPath(), filePath, language, SettingsManager::get(QString("%1/Run Command").arg(language)).toString(),
              SettingsManager::get(QString("%1/Run Arguments").arg(language)).toString(), testcases->input(index),
-             SettingsHelper::getTimeLimit());
+             timeLimit());
```

Comment:
Now that every tab has its own timelimit. We can add another verdict "TLE" and also pull timelimit for the tab from Competitive Companion

Reply:
> We can add another verdict "TLE"

That worths another PR. Though different time limits for different tabs make the TLE verdict more useful, it's not really related to this. i.e. this PR works without the TLE verdict and the TLE verdict without this PR. I can open another PR for the TLE verdict without conflicts with this PR.